smalltalksrc/Slang/CCodeGenerator.class.st

```diff
@@ -233,6 +233,7 @@ CCodeGenerator >> addMethodFor: aClass selector: selector [
 
 	| method tmethod |
 	method := aClass compiledMethodAt: selector.
+	(method pragmaAt: #halt) ifNotNil: [ 1halt ].
```

Comment:
haha interesting. I like the idea. Could we rename it later as something more slangish?
Like `<slangHalt>` ? Also, maybe we should have different halts?

```smalltalk
<slangHalt: inline>
<slangHalt: init>
<slangHalt: codeGeneration>
```

....